### ISSUE TITLE
Primary getting in bootloop or fastboot

### ISSUE THREAD

**Issue by phitschhd** [vasishath/multirom#30]
I flashed Pixel Experience 11 in TWRP internal (primary) without magisk then multirom.zip and multirom recovery. When i reboot it boots automatically (without multirom menu) in internal, then i created a pattern in the rom and at the secondary reboot multirom asks for the pattern after successfully decryption pixel experience getting in bootloop. Then i flashed multirom.zip again, because in the Multirom TWRP stands multirom not installed then it boots in fastboot. I also tried lineageos 18 but it also runs after install of multirom.zip in fastboot mode. :(

**Comment by vasishath** [comment]
Is this happening with the latest build ?

**Comment by phitschhd** [comment]
yes with the latest build. secondary rom also getting in bootloop. (Pixel Experience 11)

**Comment by vasishath** [comment]
Wait.. pixel experience is primary or secondary ?

**Comment by phitschhd** [comment]
i have tried pixel experience at primary and also secondary. in primary pixel experience only works when i flash pixel experience, then magisk and after i setup all in the rom, i installed multirom.zip and multirom twrp. when i not flash magsik on primary it getting in fastboot mode or in bootloop.

**Comment by vasishath** [comment]
Ok so the issue u described with secondary also doesn't occur if u have flashed magisk on primary ?

**Comment by phitschhd** [comment]
i have not tried to install magisk on secondary.. work in the new version of multirom magsik v22?

**Comment by phitschhd** [comment]
in secondary it only gets in bootloop. the fastboot problem seems only be at primary

**Comment by vasishath** [comment]
Don't install magisk on secondary.. it wipes storage.. has happened thrice with me.

**Comment by vasishath** [comment]
Are u able to boot any secondary rom ? Miui or any

**Comment by phitschhd** [comment]
oh sh** that’s frustrating :/ 
have to try it, i only tried pixel exp. do you know the reason why it getting in fastboot on primary? it also was a problem on the previous version from multirom.

**Comment by vasishath** [comment]
Try using miui as secondary.. if it fails then it will be final that no rom is booting. For fastboot issue, I will have to check

**Comment by phitschhd** [comment]
oook. thanks for your help and for active development!! :)) very cool that you work om this tool.

**Comment by phitschhd** [comment]
i have good news - booted successfully arrow os 11 on secondary! :)

**Comment by raisaab198** [comment]
Hi Vasishath, Firstly, thank you very much for providing us support for the legendary utility. But unfortunately, I am having the same issue with my device as it stuck at fastboot mode after flashing multirom.zip file. You will find details below:
1. Pre flashing details of device:::>
a. Running TWRP version-> 3.4.0
b. Running ROM-> crDroid 7.9 Vanilla build 8th sept 2021, android 11 based

2. Steps I have followed::::>
a. Reboot into twrp and flash multirom.zip then recovery.img and seccessfully rebooted into twrp again.
b. Now, when I tried to boot into system, then it goes to fastboot mode after showing me multirom selection screen.
c. Then reboot into recovery again and add the same crdroid rom as external and it booted successfully as external rom. But internal rom goes to fastboot mode.
d. Then I wiped everything from twrp and did a clean installation of AOSiP and the folloed the same steps to flash multirom again and again tried to boot my device and this time it goes to bootloop.

The problem is only with primary rom I guess. Or maybe with android 11 based roms. But hope in future, it will be solved

**Comment by Rodrigoteckcell** [comment]
OLA BOA TARDE ESTOU COM PROBEMAS EM USAR . APOS O FLASH DO multirom.zip ENTRA EM MODO FASTBOOT
